.vscode/tasks.json

```
@@ -16,6 +16,9 @@
 			"label": "make",
 			"type": "shell",
 			"command": "make",
+			"args": [
+				"--jobs=16"
```

We use 8 in other places, so that is more consistent
```suggestion
				"--jobs=8"
```